cmd/bucket-lifecycle-handlers.go

```diff
@@ -86,6 +91,11 @@ func (api objectAPIHandlers) PutBucketLifecycleHandler(w http.ResponseWriter, r
 		return
 	}
 
+	if bucketLifecycle.HasExpiry() || (lcCfg != nil && *lcCfg.ExpiryRuleRemoved) {
```

Comment:
```suggestion
	if bucketLifecycle.HasExpiry() || (lcCfg != nil && lcCfg.ExpiryRuleRemoved != nil && *lcCfg.ExpiryRuleRemoved) {
```
ExpiryRuleRemoved is also a pointer - can you rename this to ExpiryRuleRemovedPtr